Unix/pal/alloc.c

```diff
@@ -119,16 +120,12 @@ void PAL_DumpAllocList()
     if (!_list)
         return;
 
-    printf("\nWARNING: one or more blocks still allocated!\n");
```

Comment:
I would be tempted to leave the printf as well.

Reply:
This would duplicate the output if omiserver is run with -l.